lib/vmfloaty.rb

```diff
@@ -562,6 +562,25 @@ def run
       end
     end
 
+    command :completion do |c|
+      c.syntax = 'floaty completion [options]'
+      c.summary = 'Outputs path to completion script'
+      c.description = "Outputs path to a completion script for the specified shell, or bash if not specified.\nWill exit non-zero if no completion script is available for the requested shell."
+      c.example 'Gets path to bash tab completion script', 'floaty completion --shell bash'
+      c.option '--shell STRING', String, 'Shell to request completion script for'
+      c.action do |args, options|
+        shell = (options.shell || 'bash').downcase.strip
+        completion_file = File.expand_path(File.join('..', '..', 'extras', 'completions', "floaty.#{shell}"), __FILE__)
+
+        if File.exist?(completion_file)
+          puts completion_file
+          exit 0
+        else
+          exit 1
```

Comment:
Does it make sense here to print a warning to STDERR in the case where the file requested is missing?